WebPages/productListing/productlist.css

```diff
@@ -0,0 +1,107 @@
+@import url(/main.css);
+@import url(/index.css);
+
+.nav-container{
+    box-shadow: var(--shadow-box);
+}
+
+.flex-space-between{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+
+.filter-and-category{
+display: flex;
+flex-direction: column;
+ grid-area: scroller;
+overflow-y: auto;
+overflow-x: auto;
```

Comment:
no need to define both if you want overflow-x and y just write **overflow : auto**

Reply:
Done , can you approve now